onboarded.txt

```diff
@@ -3,3 +3,4 @@ lucyparsonslabs.com,lucyparsonslabs.securedrop.tor.onion
 theintercept.com,theintercept.securedrop.tor.onion
 www.theguardian.com,theguardian.securedrop.tor.onion
 www.abc.net.au,abc.au.securedrop.tor.onion
+www.hmdc.harvard.edu,iqss.harvard.securedrop.tor.onion
```

Comment:
Note that the organization in question has its homepage at https://www.iq.harvard.edu/ . We're still waiting for https://github.com/freedomofpress/securedrop.org/pull/761 to land; once it does, we may want to update the organization URL for the directory entry, and here as well.